spanner_orm/tests/migrations_for_emulator_test/create_custom_length_field_f959b767457d.py

```diff
@@ -0,0 +1,33 @@
+"""Spanner ORM migration: create_custom_length_field.
+
+Migration ID: 'f959b767457d'
+Created: 2022-09-13 13:28:34-07:00
+"""
+
+import spanner_orm
+
+migration_id = 'f959b767457d'
+prev_migration_id = '69a8f072dacf'
+
+
+class OriginalTeeTable(spanner_orm.model.Model):
+  """ORM Model with the original schema for the Commands table.
+  Don't update this model, create new migrations instead.
+  """
+
+  __table__ = 'Tee'
+  id = spanner_orm.Field(spanner_orm.String, primary_key=True)
+  cus_str = spanner_orm.Field(spanner_orm.String(20))
+  cus_arr_str = spanner_orm.Field(spanner_orm.Array(spanner_orm.String(4)))
+  cus_byt = spanner_orm.Field(spanner_orm.BytesBase64(20))
+  cus_arr_bye = spanner_orm.Field(spanner_orm.Array(spanner_orm.BytesBase64(4)))
```

Comment:
(nit) "Function names, variable names, and filenames should be descriptive; eschew abbreviation. In particular, do not use abbreviations that are ambiguous or unfamiliar to readers outside your project, and do not abbreviate by deleting letters within a word." https://google.github.io/styleguide/pyguide.html#316-naming

Reply:
Done.